printer_mods/edwardyeeks/Decontaminator_Purge_Bucket_&_Nozzle_Scrubber/Macros/nozzle_scrub.cfg

```diff
@@ -161,7 +161,7 @@ gcode:
          {% endif %}
 
          ### Position for purge. Randomly selects middle of left or right bucket. It references from the middle of the left bucket.
-         G1 X{bucket_start + (bucket_left_width / 2) + (bucket_pos * bucket_gap) + (bucket_pos * (bucket_right_width / 2))}
+         G1 X{bucket_start + (bucket_left_width / (2 - bucket_pos) + (bucket_pos * bucket_gap) + (bucket_pos * (bucket_right_width / 2))}
```

Comment:
The brackets are not balanced causing syntax error

Reply:
I think it should be:
`G1 X{bucket_start + (bucket_left_width / (2 - bucket_pos)) + (bucket_pos * bucket_gap) + (bucket_pos * (bucket_right_width / 2))}`

Reply:
You're right, thanks for picking it up!